components/_patterns/02-molecules/address-bar/partials/date-bar/_date-bar.twig

```diff
@@ -40,7 +40,7 @@
   {# Range Not Yet Open #}
   {% elseif (event_date_start|date(time__ymd) > now) and (event_date_end|date(time__ymd) > event_date_start|date(time__ymd)) %}
     <div>
-      Open:
+      Opens:
```

Comment:
<img width="1292" alt="Screen Shot 2019-09-09 at 4 35 36 PM" src="https://user-images.githubusercontent.com/1720093/64564591-e1f6b700-d31f-11e9-835d-ccc9514ee643.png">

@jackkim9 this variation refers to an event positioned happening in the future that has not yet begun. Brian's initial pass at this is seen above, but I imagine you may have thoughts on how you'd like us to tackle this variation.

Reply:
A way to bring this to close to the current design would be to reformat it as:

Opening Soon
{Month} {DD}–{Month} {DD}

Reply:
I made this update. We can always revisit if design specs change/update.